### DISCUSSION TITLE
Camera2 with fpfc

### DISCUSSION THREAD

**Discussion by PD-Colt** [kinsi55/CS_BeatSaber_Camera2#39] [Q&A] [answered]
So I am trying to record a beat saber replay with only my pc because my headset broke. I enabled fpfc on steam and launched beat saber. I installed camera2, created a new scene and then went to my files and copied and pasted a camera 2 preset for a certain camera angle. In that scene I turned on fpfc, replay, and playing multi mode. In a replay it is still first person mode and no camera angle shown on the screen. Any way I can fix this?

Here is what I'm using:

`{
	"antiAliasing":2,
	"farZ":1000,
	"FOV":80,
	"FPSLimiter":{
		"fpsLimit":0
	},
	"layer":1000,
	"ModmapExtensions":{
		"autoHideHUD":false,
		"autoOpaqueWalls":true
	},
	"orthographic":false,
	"renderScale":1.2,
	"Smoothfollow":{
		"followReplayPosition":true,
		"forceUpright":false,
		"pivotingOffset":false,
		"position":10,
		"rotation":5
	},
	"targetPos":{
		"x":0,
		"y":0,
		"z":-0.8
	},
	"targetRot":{
		"x":0,
		"y":0,
		"z":0
	},
	"type":"FirstPerson",
	"viewRect":{
		"height":-1,
		"width":-1,
		"x":0,
		"y":0
	},
	"visibleObjects":{
		"Avatar":true,
		"CutParticles":false,
		"Debris":false,
		"Floor":true,
		"Notes":"Visible",
		"UI":true,
		"Walls":"Visible"
	}
}`

**Comment by kinsi55** [comment, marked as answer]
>"type":"FirstPerson"

;)

If you want just a fixed position camera you need Positionable

**Reply by PD-Colt** [reply]
Is that a mod?

**Reply by PD-Colt** [reply]
And if so I cannot find it in the bsmg server

**Reply by kinsi55** [reply]
What, I recommend you to read the Wiki: https://github.com/kinsi55/CS_BeatSaber_Camera2/wiki

**Reply by PD-Colt** [reply]
Can I change positioned with fpfc mode?

**Reply by PD-Colt** [reply]
Also I know what positioned is I just forgot. Heh

**Reply by kinsi55** [reply]
You can move around positionable cameras in fpfc yes
